hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -282,8 +282,8 @@
       name: rom_e2e_sigverify_always_a_bad_b_bad_test_unlocked0
       uvm_test_seq: chip_sw_rom_e2e_sigverify_always_a_bad_b_bad_vseq
       sw_images: [
-        "//sw/device/silicon_creator/rom/e2e/boot_policy_valid:empty_test_slot_a_corrupted:1:ot_flash_binary:signed:fake_rsa_test_key_0",
-        "//sw/device/silicon_creator/rom/e2e/boot_policy_valid:empty_test_slot_b_corrupted:2:ot_flash_binary:signed:fake_rsa_test_key_0",
+        "//sw/device/silicon_creator/rom/e2e:empty_test_slot_a_corrupted:1:ot_flash_binary:signed:fake_rsa_test_key_0:new_rules",
```

Comment:
This isn't obvious unless you read through `//hw/top_earlgrey/dv/env/chip_env_cfg.sv`:  `ot_flash_binary` and `new_rules` have the same effect - they shed the legacy naming convention of certain binary resources being named `<something>_prog_<more-junk>`.

You don't necessarily need to change this as #19973 will get rid of `new_rules` once we finish the conversion and get rid of support for the legacy "functest" macro.

Reply:
Yes I realized that when I read the code but I figured that there is no harm in adding a redundant `new_rules` and it marks that those have been converted.